tools/istio-iptables/pkg/cmd/root.go

```diff
@@ -379,13 +369,9 @@ func bindCmdlineFlags(rootCmd *cobra.Command) {
 
 	rootCmd.Flags().Bool(constants.CNIMode, false, "Whether to run as CNI plugin.")
 
-	rootCmd.Flags().Bool(constants.HostNSEnterExec, false, "Instead of using the internal go netns, use the nsenter command for switching network namespaces.")
+	rootCmd.Flags().String(constants.IptablesVersion, "", "version of iptables command. If not set, this is automatically detected.")
```

Comment:
what is the benefit of passing the version? 
it can cause subtle bugs if there is a mismatch, and in case you want to verify that, you can just get the version directly

Reply:
The main motivation was just if there was some weird bug or non-standard environment that needed to be worked around. Not sure its really needed.

Reply:
+1 removing it unless we know we need it.